lib/modules/@apostrophecms/ui/src/apos/components/AposAreaMenuItem.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <button
+    @click="click" class="apos-area-menu__button"
+    :data-action="item.action"
+    :tabindex="String(tabindex)"
+    @keydown.prevent.arrow-down="$emit('down')"
+    @keydown.prevent.arrow-up="$emit('up')"
+  >
+    <component
+      v-if="item.icon"
+      :size="15"
+      class="apos-area-menu__item-icon"
+      :is="item.icon"
+    />
+    {{ item.label }}
+  </button>
+</template>
+
+<script>
+
+export default {
+  props: {
+    item: {
+      required: true,
+      type: Object
+    },
+    tabbable: {
+      type: Boolean,
+      default: true
+    }
+  },
+  emits: ['click', 'up', 'down'],
```

Comment:
A Vue 3 ism, but doesn't seem to hurt in Vue 2, so let's start getting ready I guess!

Reply:
The linter yells about it. The alternative would be to change the rule to disable the requirement.